modules/ts/misc/perf_tests_timing.py

```diff
@@ -1,4 +1,22 @@
 #!/usr/bin/env python
+""" Prints total execution time and number of total/failed tests.
+
+This script uses XML test log to produce basic runtime statistics in a text or HTML table.
```

Comment:
I propose to add some words on `--gtest_ouput` usage to highlight the source of xml and statistics.

Reply:
Done.